### PR TITLE
Split off repo preparation from zdup

### PR DESCRIPTION
**Pull request by lnussel**
Split off repo preparation from zdup to prepare for #5819

### CODE REVIEW

**Comment by asdil12** [comment]
Could you provide a verification run?

File: tests/installation/zdup_prepare_repos.pm [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (okurz) [inline tests/installation/zdup_prepare_repos.pm]: I think this should be 2018 only

File: tests/installation/zdup_prepare_repos.pm [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: prepare repos for zdup.test
+# Maintainer: Oliver Kurz <okurz@suse.de>
```
Comment (okurz) [inline tests/installation/zdup_prepare_repos.pm]: Not sure if I am a good maintainer but if you do not want to take it then ok :)

**Comment by foursixnine** [comment]
@lnussel are you planning on providing a verification run? or we test in production? :D

**Comment by OleksandrOrlov** [comment]
Closing the PR as there is no recent activity.

Please feel free to open new PR with resolved conflicts if you are planning to continue working on this. Thank you.

**Pull request closed without merging**